guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -68,39 +74,70 @@ If you lose connection to the command shell where the upgrade command is running
 ----
 # {foreman-maintain} upgrade run --target-version {TargetVersionMaintainUpgrade}
 ----
-
+endif::[]
+ifdef::katello[]
+. Check for running tasks
++
+[options="nowrap" subs="attributes"]
+----
+# foreman-rake katello:upgrade_check
+----
+. Update operating system packages
++
+[options="nowrap" subs="attributes"]
+----
+# yum update -y
+----
+. Update repositories
++
+.For Centos 7 or Red Hat Enterprise Linux Users:
+[options="nowrap" subs="attributes"]
+----
+# yum update -y https://fedorapeople.org/groups/katello/releases/yum/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
+# yum install -y centos-release-scl-rh
+----
+. Update packages
+Clean the yum cache and update required packages
++
+[options="nowrap" subs="attributes"]
+----
+# yum clean all
+# yum -y update
+----
+. Run installer
++
+[options="nowrap" subs="attributes"]
+----
+# {foreman-installer}
+----
+endif::[]
 . Check when the kernel packages were last updated:
 +
 [options="nowrap"]
 ----
 # rpm -qa --last | grep kernel
 ----
-
 . Optional: If a kernel update occurred since the last reboot, stop the `{foreman-maintain}` services and reboot the system:
 +
 ----
 # {foreman-maintain} service stop
 # reboot
 ----
-
 . If using a BASH shell, after a successful or failed upgrade, enter:
 +
 [options="nowrap"]
 ----
 # hash -d {foreman-maintain} service 2> /dev/null
 ----
-
 . Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups that you make.
-
 . If you make changes in the previous step, restart the `{foreman-maintain}` services.
 +
 [options="nowrap"]
 ----
 # {foreman-maintain} service restart
 ----
-
 . If you have the OpenSCAP plug-in installed, but do not have the default OpenSCAP content available, enter the following command:
```

Comment:
When would this happen? I don't know OpenSCAP but is this something seeding should take care of? If so, IMHO it can indicate a bug that we have to document this.

Reply:
i'm deleting this, as i don't understand why its needed